### ISSUE TITLE
iD forces users to add misspelled tag 'clothers'

### ISSUE THREAD

**Issue by mueschel** [openstreetmap/iD#7924]
When editing clothing shops, iD automatically adds the tag 'clothers' - which is clearly wrong.
E.g. https://www.openstreetmap.org/node/412906795/history

**Comment by quincylvania** [comment]
This looks like an error in the [name-suggestion-index](https://github.com/osmlab/name-suggestion-index) that's already been fixed there. I'll leave this open until the fix has been incorporated by iD.

**Comment by Cj-Malone** [comment]
Once it lands it might be a good idea to make sure everything gets cleaned up. [overpass](https://overpass-turbo.eu/s/YN2)

**Comment by peternewman** [comment]
On a sort of similar note, is it worth looking for and flagging some obvious key and value typos in iD e.g. `sepErate` which I often do until I catch myself and fix it?

If you e.g. ignored the value of certain fields which are supposed to be free text (e.g. name, wikipedia) I don't think there would be a huge risk of stopping people writing what really exists, but it would offer the option to avoid some typos.

E.g. https://overpass-turbo.eu/s/Zmm

Let me know if you'd rather have a new issue for this.

**Comment by quincylvania** [comment]
> Let me know if you'd rather have a new issue for this.

If think you already opened this as #7754?

**Comment by peternewman** [comment]
Oops, well at least I'm consistent! :laughing:

**Comment by quincylvania** [comment]
Closing this in favor of #8237, which is more specific to the solution.
